### PR TITLE
Revert "Make xkcd title a clickable link"

### PR DESCRIPTION
**Pull request by TheDevFreak**
Reverts rHomelab/LabBot-Cogs#187

not actually a feature discord supports in embed titles.

### CODE REVIEW

**Comment by tigattack** [comment]
Discord moment. Maybe it could be added as a field?

File: xkcd/xkcd.py [hunk 1]
```diff
@@ -41,10 +41,7 @@ async def xkcd(self, ctx: commands.Context, comic_number: int = 0):
 
     async def make_comic_embed(self, ctx: commands.Context, data: dict) -> discord.Embed:
         """Generate embed for xkcd comic"""
-        xkcd_embed = discord.Embed(
-            title=f"[xkcd Comic: #{data['num']}](https://xkcd.com/{data['num']})",
-            colour=await ctx.embed_colour()
-        )
+        xkcd_embed = discord.Embed(title=f"xkcd Comic: #{data['num']}", colour=await ctx.embed_colour())
```
Comment (issy) [inline xkcd/xkcd.py:44]:
```suggestion
        xkcd_embed = discord.Embed(
            title=f"xkcd Comic: #{data['num']}",
            url=f"https://xkcd.com/{data['num']}",
            colour=await ctx.embed_colour()
        )
```

**Review by issy — CHANGES_REQUESTED** [review]
We can use `discord.Embed(url=)` instead

**Review by issy — APPROVED** [review]
Whoever wrote the code for this PR is really talented 👍

**Pull request merged**